docs/notebooks/example1-with-sensors.xml

```diff
@@ -223,6 +223,27 @@
                     <LinkedTimeSeriesID IDref="TS-ResourceUse-Electricity-Energy-11"/>
                     <LinkedTimeSeriesID IDref="TS-ResourceUse-Electricity-Energy-12"/>
                   </AnnualFuelUseLinkedTimeSeriesIDs>
+                  <Emissions>
+                    <Emission>
+                      <EmissionBoundary>Indirect</EmissionBoundary>
+                      <EmissionsType>CO2e</EmissionsType>
+                      <GHGEmissions>TOTAL FROM OUR TIME SERIES</GHGEmissions>
```

Comment:
update with the real number in the time series

Reply:
I also made the Greenhouse Gas Emissions change above as well.